packages/DataGrid/stories/DataGrid.resize.stories.js

```diff
@@ -0,0 +1,153 @@
+        <DataGrid data={data}>
+          <DataGrid.ColumnDefinition header="Countries (grows)" cell="country" canGrow />
+          <DataGrid.ColumnDefinition header="Name" cell="name" />
+          <DataGrid.ColumnDefinition header="Name" cell="name" />
+          <DataGrid.ColumnDefinition header="Name" cell="name" />
+          <DataGrid.ColumnDefinition header="Name" cell="name" />
+          <DataGrid.ColumnDefinition header="Goals (grows)" cell="goals" canGrow />
+          <DataGrid.ColumnDefinition header="Goals" cell="goals" />
+        </DataGrid>
+      </Sbook.Story>
+    </React.Fragment>
+  );
+}
+
+function AppWithActionBar() {
+  const { orderedColumnIds, isColumnHidden, ...handlers } = useColumnsArrangement([
+    "goals",
+    "name",
+    "status",
+    "country",
+    "joined",
+  ]);
+
+  const renderColumns = type => {
+    const column = {
+      goals: {
+        cell: () => "Goals",
+        header: "Goals (grows)",
+        canGrow: true,
+      },
+      name: {
+        cell: () => "Name",
+        header: "Name",
+        canGrow: false,
+      },
+      status: {
+        cell: () => "Status",
+        header: "Status",
+        canGrow: false,
+      },
+      country: {
+        cell: () => "Country",
+        header: "Country",
+        canGrow: false,
+      },
+      joined: {
+        cell: () => "Joined",
+        header: "Joined",
+        canGrow: false,
+      },
+    };
+
+    return column[type];
+  };
+
+  return (
+    <React.Fragment>
+      <Sbook.Story>
+        <Heading level={2}>DataGrid with ActionBar</Heading>
+        <ActionBar>
+          <ColumnsArrangement orderedColumnIds={orderedColumnIds} {...handlers}>
+            <ColumnsArrangement.ColumnDefinition
+              id="goals"
+              label="Goals"
+              isDisabled={false}
+              isHidden={isColumnHidden("goals")}
+            />
+            <ColumnsArrangement.ColumnDefinition
+              id="name"
+              label="Name"
+              isDisabled={false}
+              isHidden={isColumnHidden("name")}
+            />
+            <ColumnsArrangement.ColumnDefinition
+              id="status"
+              label="Status"
+              isDisabled={false}
+              isHidden={isColumnHidden("status")}
+            />
+            <ColumnsArrangement.ColumnDefinition
+              id="country"
+              label="Country"
+              isDisabled={false}
+              isHidden={isColumnHidden("country")}
+            />
+            <ColumnsArrangement.ColumnDefinition
+              id="joined"
+              label="Joined"
+              isDisabled={false}
+              isHidden={isColumnHidden("joined")}
+            />
+          </ColumnsArrangement>
+        </ActionBar>
+        <DataGrid key={orderedColumnIds.join("-")} data={data} width={1000}>
+          {orderedColumnIds.map(columnKey =>
+            isColumnHidden(columnKey) ? null : (
```

Comment:
Suggestion, you dont have to take it:

`!isColumnHidden(columnKey) && (<DataGrid.ColumnDefinition ...`